Content/Application/ResourceLoader/ResourceLoaderInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContentBundle\Content\Application\ResourceLoader;
+
+interface ResourceLoaderInterface
+{
+    /**
+     * @param array<int|string> $ids
+     * @param mixed[] $params
+     *
+     * @return array<int|string, object> index must be the ID of the object
```

Comment:
```suggestion
     * @return array<int|string, mixed> index must be the ID of the object
```

The tag selection only returns the tag name and I think if somebody wants to return just a optimized array it should be possible so I would set this to mixed.